user_docs/en/userGuide.t2t

```diff
@@ -1665,8 +1667,8 @@ This button opens the directory where you can place custom code while developing
 This button is only enabled if NVDA is configured to enable loading custom code from the Developer Scratchpad Directory.
  
 ==== Use UI automation to access Microsoft Word document controls when available ====[AdvancedSettingsUseUiaForWord]
-When this option is enabled, NVDA will try to use  the Microsoft UI Automation accessibility api in order to fetch information from Microsoft Word document controls.
-This includes in Microsoft Word itself, and also the Microsoft Outlook message viewer and composer.
+When this option is enabled, NVDA will try to use  the Microsoft UI Automation accessibility API in order to fetch information from Microsoft Word document controls.

```

Comment:
There is a double space after "NVDA will try to use" might as well fix it while changing this line.